### ISSUE TITLE
Missing tabs objects

### ISSUE THREAD

**Issue by jakobnotseth** [microsoftgraph/msgraph-sdk-powershell#238]
Graph API does not return tabs objects when listing a Teams teams channels tabs.
GET https://graph.microsoft.com/v1.0/teams/{id}/channels/{id}/tabs?$expand=teamsApp
Documented here https://docs.microsoft.com/en-us/graph/api/teamstab-list?view=graph-rest-1.0
Expected result - list of tabs
Actual result - empty value

**Comment by jakobnotseth** [comment]
and sorry if this got in the wrong repo, finding where you report core issues with Graph is impossible

**Comment by darrelmiller** [comment]
@jakobnotseth I understand it is challenging to know where to report these issues.  We have a support page here that attempts to direct you as to the best place.  https://developer.microsoft.com/en-us/graph/support  

Also, I just tried using [Graph Explorer](https://developer.microsoft.com/en-us/graph/graph-explorer) and I was able to get the API to return me a list of tabs using the query you mentioned.
